tests/plugins/image2/editing.js

```diff
@@ -74,6 +74,37 @@
 			wait();
 		},
 
+		// #1348
```

Comment:
Please remember about proper issues referencing (see https://docs.ckeditor.com/ckeditor4/latest/guide/dev_tests.html#tagging-tests). Here should be:

> // (#1348)